geo/benches/geodesic_distance.rs

```diff
@@ -0,0 +1,19 @@
+#[macro_use]
+extern crate criterion;
+extern crate geo;
+
+use geo::prelude::*;
+
+fn criterion_benchmark(c: &mut criterion::Criterion) {
+    c.bench_function("geodesic distance f64", |bencher| {
+        let a = geo::Point::<f64>::new(17.107558, 48.148636);
+        let b = geo::Point::<f64>::new(16.372477, 48.208810);
+
+        bencher.iter(|| {
+            let _ = a.geodesic_distance(&b);
```

Comment:
The geodesic computation is 2-3x slower than vincenty

```
geodesic distance f64   time:   [873.59 ns 896.28 ns 922.24 ns]                                   
vincenty distance f64   time:   [357.93 ns 359.94 ns 361.92 ns]                                  
```

Reply:
FYI we can get this down 20% if we're willing to make a hard dependency on the `lazy_static` crate (we already have several transitive dependencies on it). Shown here:  https://github.com/michaelkirk/rust-geo/commit/7e2c22c91b019f032afc9d23b9858f83b6b01b1e

geodesic distance f64   time:   [720.97 ns 725.92 ns 732.69 ns]

Reply:
would `lazy_static` be added to this crate or the `geodesiclib` crate? + 1 from me either way

Reply:
I was originally thinking here in geo, but now that you mention it, it makes more sense in the geographiclib crate so any other potential users could benefit.

Reply:
Done in https://github.com/michaelkirk/geographiclib-rs/pull/4